source/reference/options.txt

```diff
@@ -468,6 +474,40 @@ Authentication Options
    - ``none``, the effect is the same as setting
      ``authMechanismProperties=CANONICALIZE_HOST_NAME:false``.
 
+.. option:: --oidcFlows
+
+   Specifies OpenID Connect flows. The OpenID connect flows specify how the ``mongosh``
+   interacts with identity provider for the authentication process. ``mongosh`` 
+   supports the following OpenID Connect flows:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 25 75
+
+      * - OpenID Connect Flow
+        - Description
+
+      * - ``auth-code``
+        - Default. ``mongosh`` redirects you to the identity provider log-in screen.
+
+      * - ``device-auth``
+        - ``mongosh`` provides you with a URL and code to finish authentication.    
+
+.. option:: --oidcRedirectUri
+
+   Specifies a URI where the identity provider redirects you after authentication.
+   The default is ``http://localhost:27097/redirect``.
+
+.. option:: --oidcTrustedEndpoint
+
+   Specifies a connection to a trusted endpoint that is not Atlas or localhost. 
+   Only connect to hosts that you know are safe. 
+
+.. option:: --browser
+
+   Specifies the specific browser ``mongosh`` redirects you to when ``MONGODB-OIDC`` 
+   is enabled. 
```

Comment:
We should mention that this is command will be run with the system shell (e.g. bash, zsh, etc.).

Reply:
Done.